src/components/PluginReadableInstalls.jsx

```diff
@@ -5,7 +5,7 @@ function PluginReadableInstalls({currentInstalls}) {
     if (!currentInstalls) {
         return <>No usage data available</>;
     }
-    return <>{currentInstalls}</>;
+    return <>{new Intl.NumberFormat('en-US').format(currentInstalls)}</>;
```

Comment:
I've never used this util. If you specify the format does it only do it for us style numbers? Is possible to let the browser pick the locale?

Reply:
If we omit the locale name, the browser locale will be used. But when formatting dates we are also using US locale and [almost](https://plugins.jenkins.io/redis-notifier/) all the texts are in English, so I thought this is more consistent.

Reply:
Oh yea totally a good plan. I was just curious how it worked.

I think I started separating dates into its own components for that reason, so we could eventually fix it. I just didn't go far enough